README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img alt="capi" src="https://github.com/kubernetes-sigs/cluster-api/raw/main/docs/book/src/images/introduction.png" width="160x" /><img alt="capi" src="https://cloud.google.com/_static/cloud/images/favicons/onecloud/super_cloud.png" width="192x" /></p>
+<p align="center"><img alt="capi" src="https://github.com/kubernetes-sigs/cluster-api/raw/main/docs/book/src/images/introduction.png" width="800x" height="300x" /></p>
```

Comment:
can you check the rendered page, for looks like was a bit off and weird

Reply:
Sure 
It is fine I think any thing specifically which you noticed that is not good?

Reply:
the capi logo is weird
![Screenshot 2022-03-05 at 15 55 54](https://user-images.githubusercontent.com/4115580/156888658-8390c3bf-95c7-4821-bf1f-cdcc854a5ef4.png)

Reply:
@cpanato Check now?